x-pack/plugins/event_log/server/es/cluster_client_adapter.test.ts

```diff
@@ -15,20 +15,21 @@ import { contextMock } from './context.mock';
 import { findOptionsSchema } from '../event_log_client';
 import { delay } from '../lib/delay';
 import { times } from 'lodash';
+import { DeeplyMockedKeys } from 'packages/kbn-utility-types/target/jest';
```

Comment:
**nit:** I see there is no example on how to require this via a package. I found a way below, which seems better.

```suggestion
import { DeeplyMockedKeys } from '@kbn/utility-types/jest';
```